homeassistant/components/sensor/wunderground.py

```diff
@@ -679,12 +676,10 @@ def __init__(self, hass: HomeAssistantType, rest, condition,
         self._entity_picture = None
         self._unit_of_measurement = self._cfg_expand("unit_of_measurement")
         self.rest.request_feature(SENSOR_TYPES[condition].feature)
-        current_ids = set(hass.states.async_entity_ids(sensor.DOMAIN))
-        current_ids |= hass.data[ADDED_ENTITY_IDS_KEY]
-        self.entity_id = async_generate_entity_id(
-            ENTITY_ID_FORMAT, "{} {}".format(namespace, condition),
-            current_ids=current_ids)
-        hass.data[ADDED_ENTITY_IDS_KEY].add(self.entity_id)
+        # This is only the suggested entity id, it might get changed by
+        # the entity registry later.
+        self.entity_id = sensor.ENTITY_ID_FORMAT.format('pws_' + condition)
```

Comment:
So I am not 100% sure if a suggested entity ID is even needed when we have a unique ID? Because if a person configures one entity, wouldn't they want it to be just called after the condition? Home Assistant is already making sure that they are getting unique IDs so we might as well name them as obvious as possible.

Reply:
You're suggesting something like `sensor.ENTITY_ID_FORMAT.format(condition)`?

This mostly here to allow for a seamless (backwards compatible) transition to the new unique_id system. Because without this suggestion everybody's entity IDs would break (previously they always were `sensor.pws_<condition>`)

I mean we could make this a breaking change and force people to manually change the entity ids of their sensors but really I'd rather not.